src/zarr/core/buffer/cpu.py

```diff
@@ -154,7 +154,7 @@ def create(
         order: Literal["C", "F"] = "C",
         fill_value: Any | None = None,
     ) -> Self:
-        ret = cls(np.empty(shape=tuple(shape), dtype=dtype, order=order))
+        ret = cls(np.zeros(shape=tuple(shape), dtype=dtype, order=order))
         if fill_value is not None:
             ret.fill(fill_value)
```

Comment:
```suggestion
        ret = cls(np.full(shape=tuple(shape), fill_value=fill_value or 0, dtype=dtype, order=order))
```

Untested but I'm fairly sure this pans out to be the same thing and is probably a bit faster.